### ISSUE TITLE
[A11y] Timeline show page

### ISSUE THREAD

**Issue by Dananji** [avalonmediasystem/avalon#6421]
### Description

From the accessibility checks performed on timeline related pages in https://github.com/avalonmediasystem/avalon/issues/6314, the following issue was identified for the timeline show page.

| Success criterion | WCAG 2.1 Level | Accessibility checker | Error | Relevant HTML element |
| --- | --- | --- | --- | --- |
| 4.1.2 Name, Role, Value | Level A | SiteImprove & axeDevTools | Inline frame missing a text alternative | iframe used to render timeline |

### Done Looks Like
- [ ] iframe has a text alternative that can be read by screen readers and other assistive technology tools

**Comment by joncameron** [comment]
This is no longer flagged by SiteImprove 👍
